test/RBAC.test.js

```diff
@@ -45,17 +45,17 @@ contract('RBAC', function (accounts) {
         .should.be.fulfilled;
     });
     it('does not allow admins to call #nobodyCanDoThis', async () => {
-      expectThrow(
+      await expectThrow(
```

Comment:
@Shrugs we missed this when we merged it into the repo.

Reply:
😭